apps/heft/package.json

```diff
@@ -23,6 +23,8 @@
     "@rushstack/node-core-library": "3.24.3",
     "@rushstack/ts-command-line": "4.4.4",
     "@types/tapable": "1.0.5",
+    "chokidar": "3.4",
+    "glob-escape": "~0.0.1",
```

Comment:
The latest is `0.0.2`.  Whereas `~0.X.Y` is an exact match because of the leading zero.

Also... `glob-escape` was abandoned 4 years ago.  If there isn't a modern alternative, maybe we should make a `node-core-library` API.

Reply:
It's a super simple library, so I don't have a strong opinion about replacing it vs just continuing to use it. Let's tackle that in a separate PR, though.

Reply:
I saw this elsewhere:
```ts
    const results: string[] = await LegacyAdapters.convertCallbackToPromise(glob, pattern, {
      cwd: resolvedSourceFolderPath,
      nodir: true,
      ignore: exclude
    });
```

Maybe we should make a nice `Glob` API for node-core-library.

> Let's tackle that in a separate PR, though.

Absolutely